engine/access/rpc/backend/backend_test.go

```diff
@@ -721,6 +755,33 @@ func (suite *Suite) TestGetEventsForHeightRange() {
 		suite.Require().Equal(expectedResp, actualResp)
 	})
 
+	suite.Run("valid request with max_height and min_height > last_sealed_block_height", func() {
```

Comment:
a test to address the fix for issue https://github.com/onflow/flow-js-sdk/issues/491#issuecomment-770001583